tools/integration-test/src/chain/driver.rs

```diff
@@ -456,7 +456,7 @@ impl ChainDriver {
     ) -> Result<(), Error> {
         assert_eventually_succeed(
             "wallet reach expected amount",
-            20,
+            40,
```

Comment:
It seems like there is some performance regression when the relayer is setup with multiple channels. I have to increase the timeout limit for `assert_eventual_wallet_amount` to > 20s for the IBC transfer test to not fail.